cypress/integration/host/host_sidebar.ts

```diff
@@ -0,0 +1,46 @@
+const pathWithTask = `/host/i-0fb9fe0592ea3815`;
+const pathNoTask = `/host/macos-1014-68.macstadium.build.10gen`;
+
+describe("Host page title and sidebar ", () => {
+  before(() => {
+    cy.login();
+  });
+
+  beforeEach(() => {
+    cy.listenGQL();
+    cy.preserveCookies();
+  });
+
+  it("title shows the host name", () => {
+    cy.visit(pathNoTask);
+    cy.get("[data-cy=page-title").should(
+      "include.text",
+      "Host: macos-1014-68.macstadium.build.10gen"
+    );
+  });
+
+  it("Metadata card last Communication has the correct values", () => {
+    cy.visit(pathWithTask);
+    cy.dataCy("host-last-communication").contains(
+      "Last Communication: Jul 20, 2020, 3:16:36 pm"
+    );
+  });
+
+  it("Metadata card Distro and current running links should have href", () => {
+    cy.visit(pathWithTask);
+    cy.get("[data-cy=distro-link]").should("have.attr", "href");
+    cy.get("[data-cy=running-task-link]").should("have.attr", "href");
+  });
+
+  it("Metadata card Current task should display none when running task does not exist", () => {
+    cy.visit(pathNoTask);
+    cy.dataCy("current-running-task").contains("none");
+  });
+
+  it("sshCommand has the correct values", () => {
+    cy.visit(pathWithTask);
+    cy.dataCy("ssh-command").contains(
+      "ssh admin@ec2-54-146-18-248.compute-1.amazonaws.com"
+    );
+  });
+});
```

Comment:
adding a test for ensuring text is copied to clipboard will add greater value here

Reply:
Based on a brief google search it seems to be a bit complicated to write in cypress. Considering that the copy functionality is not very important to the page, and is part of a leafy-green component which I assume is well tested on their end, it probably becomes a less worthwhile pursuit. Please lmk if you disagree.